francis/__main__.py

```diff
@@ -182,7 +188,7 @@ def listen(audio_sample):
         the_model = load_model("model.h5")
 
     print("predicting ...")
-    predictions = np.around(the_model.predict(spectrograms))
+    predictions = around(the_model.predict(spectrograms))
```

Comment:
Only numpy function used in main so changed import to only import this function to reduce workload